project/plugins.sbt

```diff
@@ -1,9 +1,24 @@
-resolvers += Resolver.bintrayIvyRepo("hmrc", "sbt-plugin-releases")
-resolvers += Resolver.bintrayRepo("hmrc", "releases")
-resolvers += Resolver.typesafeRepo("releases")
-
-addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"     % "2.13.0")
-addSbtPlugin("uk.gov.hmrc"       % "sbt-git-versioning" % "2.2.0")
-addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables" % "2.1.0")
-addSbtPlugin("com.typesafe.play" % "sbt-plugin"         % "2.7.9")
-addSbtPlugin("org.irundaia.sbt"  % "sbt-sassify"        % "1.5.1")
+resolvers += Resolver.url("hmrc-sbt-plugin-releases", url("https://dl.bintray.com/hmrc/sbt-plugin-releases"))(Resolver.ivyStylePatterns)
+resolvers += "HMRC Releases" at "https://dl.bintray.com/hmrc/releases"
+
+resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
+
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.13.0")
+
+addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.2.0")
+
+addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
+
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.5")
+
+addSbtPlugin("org.irundaia.sbt" % "sbt-sassify" % "1.4.11")
+
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
+
+addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
+
+addSbtPlugin("net.ground5hark.sbt" % "sbt-concat" % "0.2.0")
+
+addSbtPlugin("com.typesafe.sbt" % "sbt-uglify" % "2.0.0")
+
+addSbtPlugin("com.typesafe.sbt" % "sbt-digest" % "1.1.4")
```

Comment:
would these versions be okay to have?
addSbtPlugin("com.typesafe.play" % "sbt-plugin"         % "2.7.9")
addSbtPlugin("org.irundaia.sbt"  % "sbt-sassify"        % "1.5.1")

Reply:
Done